tox.ini

```diff
@@ -216,7 +219,7 @@ commands =
     pytest {tty:--color=yes} {posargs} \
       --save-unmapped-ids \
       --ignore-foreign-key-constraints \
-      --etl-settings src/pudl/package_data/settings/etl_full.yml \
+      --etl-settings src/pudl/package_data/settings/etl_fast.yml \
```

Comment:
@jdangerx did you change this from full to fast for development purposes?

Reply:
Yeah, I did - specifically to see if it was the whole 'tables' thing that was causing the tests to not run. Then I removed the tables from etl_full, which also worked, but then we moved it to `fast` so that we wouldn't spend all day waiting for the ETL to run in our call.

Reply:
Sounds good. So the changes in etl_full.yml should get merged in but this line should change back to `etl_full.yml`?

Reply:
I went ahead and changed this back because it seems like the right thing to do